cluster/saltbase/salt/flannel-server/flannel-server.manifest

```diff
@@ -0,0 +1,99 @@
+{
+    "kind": "Pod",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "flannel-helper",
+        "namespace": "kube-system",
+        "labels": {
+            "app": "flannel-helper",
+            "version": "v0.1"
+        }
+    },
+    "spec": {
+        "volumes": [
+            {
+                "name": "varlogflannel",
+                "hostPath": {
+                    "path": "/var/log"
+                }
+            },
+            {
+                "name": "etcdstorage",
+                "emptyDir": {}
+            },
+            {
+                "name": "networkconfig",
+                "hostPath": {
+                    "path": "/etc/kubernetes/network.json"
```

Comment:
I agree a blob is the right longer-term.  My concern is only that one might
think that replacing this file might effect a change, but it won't.  OK for
now.

On Tue, Nov 24, 2015 at 1:23 PM, Prashanth B <notifications@github.com>
wrote:

> In cluster/saltbase/salt/flannel-server/flannel-server.manifest
> <https://github.com/kubernetes/kubernetes/pull/17668#discussion_r45796702>
> :
>
> > +    "spec": {
> > +        "volumes": [
> > +            {
> > +                "name": "varlogflannel",
> > +                "hostPath": {
> > +                    "path": "/var/log"
> > +                }
> > +            },
> > +            {
> > +                "name": "etcdstorage",
> > +                "emptyDir": {}
> > +            },
> > +            {
> > +                "name": "networkconfig",
> > +                "hostPath": {
> > +                    "path": "/etc/kubernetes/network.json"
>
> Or we can only update it by restarting the pod right? which is what I'd
> expect (updating the network requires at least a kube-push). ideally we
> would store this in something like a config blob I guess, if not a network
> resource.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17668/files#r45796702>.
>